xml/admin_monitoring_alerting.xml

```diff
@@ -1071,30 +1071,26 @@ monitoring:
        <term><option>MON_DOWN</option></term>
        <listitem>
          <para>One or more monitor daemons is currently down.
-           The cluster requires a majority (more than 1/2) of
```

Comment:
"more than half"?

Reply:
This is deleted, only the green remains/is edited.